README.md

```diff
@@ -125,7 +138,7 @@ Another introduction to 🤗 Datasets is the tutorial on Google Colab here:
 
 We have a very detailed step-by-step guide to add a new dataset to the ![number of datasets](https://img.shields.io/endpoint?url=https://huggingface.co/api/shields/datasets&color=brightgreen) datasets already provided on the [HuggingFace Datasets Hub](https://huggingface.co/datasets).
 
-You will find [the step-by-step guide here](https://huggingface.co/docs/datasets/share.html) to add a dataset on the Hub.
+You will find [the step-by-step guide here](https://huggingface.co/docs/datasets/upload_dataset) to upload a dataset on the Hub, and also [how to do it using git](https://huggingface.co/docs/datasets/share).
```

Comment:
Nit: second option is "using Git"; then first option is using what?
Maybe something similar to:
```suggestion
You can find:
- [how to upload a dataset to the Hub using your web browser or Python](https://huggingface.co/docs/datasets/upload_dataset) and also
- [how to upload it using Git](https://huggingface.co/docs/datasets/share).
```